internal/monitors/utilization/perfcounter/perfcounter.go

```diff
@@ -0,0 +1,322 @@
+		metricName = "cpu.utilization_per_core"
+		dimensions = map[string]string{"plugin": monitorType, "core": instance}
+	}
+
+	// parse metric value
+	var metricVal datapoint.Value
+	var err error
+	if val, ok := ms.Fields["Percent_Processor_Time"]; ok {
+		if metricVal, err = datapoint.CastMetricValue(val); err != nil {
+			errors = append(errors, err)
+			return
+		}
+		// create datapoint
+		dp := datapoint.New(metricName, dimensions, metricVal, p.metricType, time.Time{})
+		sendDatapoint(dp)
+	}
+
+	return
+}
+
+// LogicalDisk performance counter
+func LogicalDisk() PerfCounter {
+	return &LogicalDiskImpl{
+		&BasePerfCounter{
+			measurement:          "win_logical_disk",
+			includeTotal:         true,
+			warnOnMissing:        true,
+			objectName:           "LogicalDisk",
+			instanceDimensionKey: "plugin_instance",
+			metricType:           datapoint.Gauge,
+			instances:            []string{"*"},
+			counters:             []string{"% Free Space", "Free Megabytes"},
+		},
+	}
+}
+
+// NetworkInterface performance counter
+func NetworkInterface() PerfCounter {
+	return &BasePerfCounter{
+		measurement:          "win_network_interface",
+		includeTotal:         true,
+		warnOnMissing:        true,
+		objectName:           "Network Interface",
+		instanceDimensionKey: "interface",
+		metricType:           datapoint.Gauge,
+		instances:            []string{"*"},
+		counters:             []string{"Bytes Received/sec", "Bytes Sent/sec", "Packets Received Errors", "Packets Outbound Errors"},
+		metricNameMapping: map[string]string{
+			// NetworkInterface - the original collectd metrics are cumulative counters, but these gauge the current rate
+			"Bytes_Received_persec":   "network_interface.bytes_received.per_second",  // perfcounter: "Bytes Received/sec"; perfcounter reporter: ""; collectd: "if_octets.rx"
+			"Bytes_Sent_persec":       "network_interface.bytes_sent.per_second",      // perfcounter: "Bytes Received/sec"; perfcounter reporter: ""; collectd: "if_octets.tx"
+			"Packets_Received_Errors": "network_interface.errors_received.per_second", // perfcounter: ""; perfcounter reporter: ""; collectd: "if_errors.rx"
+			"Packets_Outbound_Errors": "network_interface.errors_sent.per_second",     // perfcounter: ""; perfcounter reporter: ""; collectd: "if_errors.tx"
+		},
+	}
+}
+
+// PhysicalDisk performance counter
+func PhysicalDisk() PerfCounter {
+	return &BasePerfCounter{
+		measurement:          "win_physical_disk",
+		includeTotal:         true,
+		warnOnMissing:        true,
+		objectName:           "PhysicalDisk",
+		instanceDimensionKey: "plugin_instance",
+		metricType:           datapoint.Gauge,
+		counters:             []string{"Disk Reads/sec", "Disk Writes/sec"},
+		instances:            []string{"_Total"},
+		metricNameMapping: map[string]string{
+			// perfcounter: "Disk Reads/sec"; perfcounter reporter: ""; collectd: "disk_ops.read"
+			"Disk_Reads_persec": "disk.read.per_second",
```

Comment:
This is ambiguous.  Does this mean _read operations_, _read bytes_, _read sectors_, etc?